source/extensions/filters/http/dynamo/dynamo_request_parser.cc

```diff
@@ -103,18 +103,17 @@ RequestParser::TableDescriptor RequestParser::parseTable(const std::string& oper
              TRANSACT_OPERATIONS.end()) {
     std::vector<Json::ObjectSharedPtr> transact_items =
         json_data.getObjectArray("TransactItems", true);
-    std::string next_table_name = "";
     for (const Json::ObjectSharedPtr& transact_item : transact_items) {
-      next_table_name = getTableNameFromTransactItem(*transact_item);
-      if (next_table_name.empty()) {
+      auto next_table_name = getTableNameFromTransactItem(*transact_item);
```

Comment:
convention for the envoy codebase is to explicitly add the type here. Also I would add const here as you don't expect to change the table name.

Reply:
I think it's ok to make this `const auto`.